Package.swift

```diff
@@ -23,7 +23,7 @@ import Foundation
 var kituraNetPackage: Package.Dependency
 
 if ProcessInfo.processInfo.environment["KITURA_NIO"] != nil {
-    kituraNetPackage = .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "1.0.0")
+    kituraNetPackage = .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "2.2.0")
```

Comment:
I understand why we are making this change (to pick up https://github.com/IBM-Swift/Kitura-NIO/pull/199), but we should call it out in the PR description.

Note that due to the absence of the above, this fix won't work for anyone using Kitura-NIO with Swift 4 - although I expect that to affect little to no users in practice, as early -NIO adopters will likely be on Swift 5 already.

Reply:
@djones6  I have added this in PR